apps/speed.c

```diff
@@ -508,6 +508,14 @@ static double sigs_results[MAX_SIG_NUM][3];  /* keygen, sign, verify */
 #define COND(unused_cond) (run && count < INT_MAX)
 #define COUNT(d) (count)
 
+#define TAG_LEN 16
+
+static unsigned int mode_op; /* AE Mode of operation */
```

Comment:
Could you please initialize this variable?

Reply:
This is completely unnecessary. All static variables are initialized to 0 by default.